src/events/listener.class.ts

```diff
@@ -50,12 +50,12 @@ export class Listener {
     }
 
     add() {
-        if (!this.element) throw 'No element found to add listener to';
+        if (!this.element) return;
         this.element.addEventListener(this.type, this.callback, this.options);
     }
 
     remove() {
-        if (!this.element) throw 'No element found to remove listener from';
+        if (!this.element) return;
```

Comment:
It might be good to still notify a user that there's no element. If there's no element, that might indicate another problem exists elsewhere.

Error logging will probably get an overhaul pretty soon, so this is good for now since this is a pretty critical bug atm.

Reply:
Issue #38 added for this.